pkg/core/namespace_test.go

```diff
@@ -134,7 +136,13 @@ var _ = Describe("namespace", func() {
 			mockSecrets.On("Create", mock.Anything).Run(func(args mock.Arguments) {
 				s := args[0].(*v1.Secret)
 				Expect(s.StringData).To(HaveKeyWithValue("username", "_json_key"))
-				Expect(s.StringData).To(HaveKeyWithValue("password", "{ \"project_id\": \"gcp-project-id\" }\n"))
+				var expectedPassword string
```

Comment:
That won't work since the HaveKeyWithValue matcher uses Equals internally.

Rather than redefining the value, we could wrap it in a method that replaced `\n` with `\r\n` based on the platform.